policies/communication_channels.md

```diff
@@ -0,0 +1,59 @@
+# Communication channels in the Astropy Project
+
+## Preface
+
+Communication in the Astropy Project is governed by the [Community
+Code of Conduct](https://www.astropy.org/code_of_conduct.html), which
+takes precedence over anything written here. This goal of this policy is
+to write down the technical details of how email lists, Slack
+channels, forums etc. are set up.
+
+While we aim to be an inclusive and welcoming community, we
+recognize that not all users want to be involved in the technical
+details of building and running a library like Astropy. Therefore, some
+communication channels that are more geared towards users and others
+to developers. Also, in a project this size, certain issues that
+arise require privacy (e.g. personal or financial information is
+legally protected in many contries). Thus, there are communication
+channels geared towards or exclusively for developers, maintainers, or
+members of certain committees.
+
+
+## Email communication geared towards project members
+
+Mailing lists are maintained as Google groups or through other list
+servers. Unless specifically restricted, anyone worldwide can
+subscribe to these lists, and previous posts are publicly
+available. List membership is required to post messages to prevent
+spam.  In order to be as open as possible, none of these lists shall
+be moderated and nor will posts by group members be otherwise
+restricted. However, the Coordination Committee can implement a "cool
+down" period of no more than 48 hours if they deem it necessary to
+maintain a respectful and welcoming atmosphere. During this period,
+posting may be moderated or otherwise restricted by the Coordination
+Committee. The beginning and end of such a period has to be announced
+by the Coordination Committee on the affected mailing list. The
+Coordination Committee shall not implement a "cool down" within a week
+of any scheduled Astropy vote unless in the most obvious cases of Code
+of Conduct violations.
+
+
+The following email lists / aliases are restricted in memebership and
+their content is not public:
+
+- astropy-core-maintainer: Membership: Anyone with commit rights to the github/astropy/astropy repository
```

Comment:
```suggestion
- astropy-core-maintainer: Membership: Anyone with a named role on the project
```

(I think this correction is correct but maybe @kelle @eteq @taldcroft can confirm?)

Reply:
Yes, I think that's correct. We are currently adding people to the core-maintainer list once they are nominated and accept the responsibilities associated with a named role.

Reply:
Maybe it's time then to rename the list as its name and content is in disagreement. It certainly started as a list for maintainers as at some point that had changed

Reply:
Suggestions for new name?

Reply:
astropy-core-team? astropy-team? 

Certainly not maintainers if the members are not only maintainers, the project should really stop the current practice of hijacking and override tech terms that have a well established meaning at least for those who care about the tech side of things.

Reply:
Both of those sound fine to me. Probably worth starting a new issue to get more feedback. (I think this PR can move forward with existing list name and we can update this doc if we end up changing it.)

Reply:
I'm not sure this is the "hijaking" in this case, more a evolution. In the beginning everybody involved in the project was also a core maintainer and maintainer stuff was discussed on the same list as project stuff. Then you add two more people and it's just simpler to add them to the existing list then to make a separate list. However, now we've reached a point where we may want a separate list. However, that's beyond the scope of this issue. So, I made a new one: https://github.com/astropy/astropy-project/issues/150

Reply:
The mailing list started in 2017, this was well after the project had non maintainer people in it. Otherwise I agree that this is a separate issue, and thanks for opening the separate issue.

Reply:
I'll mark that as "resolved" here since the discussion continues in #150.